addons/binding/org.openhab.binding.systeminfo.test/src/test/groovy/org/openhab/binding/systeminfo/test/SysteminfoOSGiTest.groovy

```diff
@@ -265,7 +316,15 @@ class SysteminfoOSGiTest extends OSGiTest{
         String channnelID = SysteminfoBindingConstants.CHANNEL_CPU_LOAD_1
         String acceptedItemType = "Number";
 
+        DecimalType mockCpuLoad1Value = new DecimalType(1.1)
+        when(mockedSystemInfo.getCpuLoad1()).thenReturn(mockCpuLoad1Value);
+
         initializeThingWithChannel(channnelID,acceptedItemType);
+
+        ChannelUID cpuLoad1ChannelUID = systemInfoThing.getChannel(SysteminfoBindingConstants.CHANNEL_CPU_LOAD_1).getUID()
+        SysteminfoHandler thingHandler = getThingHandler(SysteminfoHandler.class)
+
+        assertEquals(mockCpuLoad1Value, thingHandler.getInfoForChannel(cpuLoad1ChannelUID))
```

Comment:
Groovy lets you use private methods (`getInfoForChannel()`), but it is not a good practice to rely on that (we are slowly moving to pure java tests).